resources/metabase-plugin.yaml

```diff
@@ -11,19 +11,19 @@ driver:
   lazy-load: true
   parent: sql-jdbc
   connection-properties:
-    - name: client_id
-      display-name: Client ID
+    - name: user
+      display-name: User/Client ID
       required: true
-    - name: client_secret
-      display-name: Client Secret
+    - name: password
+      display-name: Password/Client Secret
```

Comment:
I believe the product decision on parameter naming was to keep the `Client ID` / `Client Secret` even in driver that support both, but add a note in V1 docs to use username/password in those fields.

Reply:
You are absolutely right but current logic required from my in jdbc driver does not allow this. I spent an hour trying to understand how to make metabase driver to support both v1 and v2 and this was the solution. Please take a look on implementation of method [getUrlVersion()](https://github.com/firebolt-db/jdbc/blob/0a4331a0d9ade4f62050266ad671f47a1f39d584/src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java#L124).

Using user/password + "old" URL format makes it possible to support both v1 and v2.

Reply:
Sorry for the confusion, I don't actually mean the parameter name or format. What I mean is, `display-name:` shouldn't have `User/` or  `Password/` prepended in the UI. Using username and password would only be described in the V1 docs, from a new user standpoint who just installs our driver there's only client id and client secret.

Reply:
I see. I can remove this but I am afraid it can confuse the existing customer(s) that (still?) use v1

Reply:
It's not something I've decided - the product team deemed it an acceptable risk so the decision was to change it.

Reply:
NP, renamed back to client_id and client_secret